package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
+    "build:styles": "sass --no-source-map ./src/styles/react-big-calendar/styles.scss ./react-big-calendar.css",
```

Comment:
Maybe `npx sass` will be enough? We don't really need sass as dependency

Reply:
agree